docs/content.zh/docs/deployment/cli.md

```diff
@@ -30,7 +30,7 @@ under the License.
 Flink provides a Command-Line Interface (CLI) `bin/flink` to run programs that 
 are packaged as JAR files and to control their execution. The CLI is part of any 
 Flink setup, available in local single node setups and in distributed setups. 
-It connects to the running JobManager specified in `conf/flink-conf.yaml`.
+It connects to the running JobManager specified in [`conf/config.yaml`]({{< ref "docs/deployment/config#flink-configuration-file" >}}).
```

Comment:
Maybe it's better to use `Flink configuration file` or `Flink 配置文件`, instead of `conf/config.yaml` or `conf/flink-conf.yaml`? So that users will not get confused if they are still using `flink-conf.yaml`.

However, it's good to add a reference to `docs/deployment/config#flink-configuration-file` for further explanation.